src/multiversioning.zig

```diff
@@ -1278,15 +1278,19 @@ pub const Multiversion = struct {
             },
             .windows => {
                 // Includes the null byte, that utf8ToUtf16LeAllocZ needs.
-                var buffer: [std.fs.max_path_bytes]u8 = undefined;
+                var buffer: [std.fs.max_path_bytes * 2]u8 = undefined;
```

Comment:
Ah, `* 2` because we use it twice. Not because of UTF-16 shenanigans, perhaps include that in the comment!